custom_components/easee/services.py

```diff
@@ -285,14 +287,23 @@ async def charger_execute_set_charging_cost(call):
             function_call = getattr(charger.site, function_name["function_call"])
             return await function_call(cost_per_kwh, vat, currency)
 
-        _LOGGER.error(
-            "Could not find charger %s", charger_id,
-        )
+        _LOGGER.error("Could not find charger %s", charger_id)
         raise HomeAssistantError("Could not find charger {}".format(charger_id))
 
+    async def charger_execute_set_access(call):
+        """Execute a service to set se cost per kwh for Easee charger site."""
```

Comment:
This comment seems like it is cut&pasted from another function and not relevant here?

Reply:
Your right. Thanks. Its fixed.